### ISSUE TITLE
https://labs.play-with-k8s.com/ is down

### ISSUE THREAD

**Issue by SophiaLG** [play-with-docker/play-with-kubernetes.github.io#54]
hi,guys, i use https://labs.play-with-k8s.com/ to create an instance, but i have try 10 times, still fail.

**Comment by leohxj** [comment]
still fail, but you visit https://www.katacoda.com/courses/kubernetes/playground to play with k8s.

**Comment by navneet2rock** [comment]
https://labs.play-with-k8s.com/  earlier it was working and session was usually valid for 8 hr. But now it's not working.

**Comment by RSwarnkar** [comment]
Yes, Not Working since 3 days. I am still awaiting this awesome site.
